packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/omnibox/ProjectsOmniboxCommandSearchService.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.omnibox;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.sirius.components.collaborative.omnibox.api.IOmniboxCommandOrderer;
+import org.eclipse.sirius.components.collaborative.omnibox.api.IProjectsOmniboxCommandProvider;
+import org.eclipse.sirius.components.collaborative.omnibox.api.IProjectsOmniboxCommandSearchService;
+import org.eclipse.sirius.components.collaborative.omnibox.dto.OmniboxCommand;
+import org.springframework.stereotype.Service;
+
+/**
+ * Used to find projects omnibox commands.
+ *
+ * @author gdaniel
+ */
+@Service
+public class ProjectsOmniboxCommandSearchService implements IProjectsOmniboxCommandSearchService {
```

Comment:
I am not really sure all these *Projects* classes belong to `sirius-component-collaborative`.
If I remember correctly, there is no _project_ concept in Sirius Components.
I don't know if I should rename these classes to remove the _project_ reference, or if I should move all this code in `sirius-web-application`.
Let me know what you think and I'll do the changes.